app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -179,7 +186,7 @@ public void onClick(View view) {
                         spinner_Team.setAdapter(adp_Team);
                         spinner_Team.setSelection(teamsInMatch.size() - 1);
                         // Set CurrentTeamToScout to be the overridden value
-                        Globals.CurrentTeamToScout = Integer.parseInt(teamNum);
+                        Globals.CurrentTeamToScout = spinner_Team.getSelectedItemPosition();
```

Comment:
getSelectedItemPosition() returns the POSITION in the list (0-6), not the text (teamnum) that is selected.  Why did you change this?  It worked before.